docs/_reporting/hubs/README.md

```diff
@@ -82,13 +95,15 @@ _<sup>2) EA billing scopes can be exported to any tenant today. Simply sign in t
 
 ## 📦 What's included
 
-The FinOps hub template includes the following resources:
+The default deployment of FinOps hub template includes the following resources:
 
 - Storage account (Data Lake Storage Gen2) to hold all cost data.
 - Data Factory instance to manage data ingestion and cleanup.
 - Key Vault to store the Data Factory system managed identity credentials.
```

Comment:
I would revert this and just add ADX to the list and call out that it's optional.
```suggestion
The FinOps hub template includes the following resources:

- Data Factory instance to manage data ingestion and cleanup.
- Storage account (Data Lake Storage Gen2) as a staging area for data ingestion.
- Azure Data Explorer (Kusto) as a scalable datastore for advanced analytics (optional).
- Key Vault to store the Data Factory system managed identity credentials.
```